.github/workflows/main.yml

```diff
@@ -95,6 +80,65 @@ jobs:
     - name: Remove reference magnification data
       run: rm -r testdata/tiff_mag_2_reference/
 
+    - name: Generate Docs
+      if: matrix.python-version == '3.7'
+      run: |
+        docs/api.sh --persist
+    
+    - name: Push docs (for branch)
+      if: matrix.python-version == '3.7'
```

Comment:
should this job check whether only a tag was pushed? or does it simply work in that case, too?

Reply:
good point!